wiggum_app/models.py

```diff
@@ -87,23 +87,25 @@ def checkSameMetadata(labeled_df, meta):
 
     return checkResult
 
-def getDistanceHeatmapDict(df):
+def getDistanceHeatmapDict(labeled_df, cur_result_df):
     """
     Generate Distance Heatmap Dictitonary List for overview 
     by grouping the results and extracting distances from result table.
 
     Parameters
     -----------
-    df : DataFrame
-        LabeledDataFrame    
+    labeled_df : LabeledDataFrame
+        object from which the cur_result_df was computed, used for meta information
+    cur_result_df : DataFrame
+        The cur_result_df parameter will be passed by filter_df, detect_df and rank_df, not always result_df
```

Comment:
```suggestion
        A result_df or derivative of it after filtering, detecting or ranking. 
```

Reply:
@Shine226 you can commit this from the PR interface, I just wanted you to see it.